.gitignore

```diff
@@ -1,3 +1,26 @@
-build
+# backup files
+*~
+*.bak
+.DS_Store
 \#*
-*~
\ No newline at end of file
+
+# project file
+*.hdp
+
+# documentation build directory
+build/
+
+# compiler build directory
+_build/
+
+# dylan tool package cache
+_packages/
+
+# package registry folder
+registry/
```

Comment:
I believe this needs to be `/registry/` so that it doesn't ignore `/sources/registry`.

Reply:
From https://git-scm.com/docs/gitignore 

> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.

Do you think is better to be more specific?

Reply:
I think using a leading separator is more explicit, which IMO is usually better if it's easy to do.